### ISSUE TITLE
500 Internal Server Error

### ISSUE THREAD

**Issue by nilskp** [polyfills/polyfills#15]
Not sure if this is the right forum, but if I try to access https://cdn.jsdelivr.net/polyfills/polyfill.js I get a 500 return status. Now this could be some temporary issues, but if I search for `polyfill` on http://www.jsdelivr.com, I get nothing.

So I wonder if the docs are outdated, or any explanation for this?

**Comment by jimaek** [comment]
The issue is with the origin server controlled by @polyfills 
It looks down or missconfigured

**Comment by nilskp** [comment]
Shouldn't the files be cached?

**Comment by jimaek** [comment]
Yes, but sooner or later they need to expire. The system cant keep it cached for ever. The cache would have worked just fine for a small to medium downtime.

**Comment by jonathanong** [comment]
i'm indisposed at the moment. i can add you as collab both here on heroku until i have time to figure it out

**Comment by jonathanong** [comment]
ohhh.... it could be heroku. i'm using the free instance and i think they changed their pricing

**Comment by jimaek** [comment]
@jonathanong http://powerful-woodland-4579.herokuapp.com/polyfill.js

**Comment by nilskp** [comment]
Ok, so that's figured out.

However, I'm a little worried about this behavior from jsDelivr, and I'll admit I don't know enough about the CDN space to point any specific fingers, but it seems to me that it's a problem if a CDN, which is suppose to be HA, is sensitive to underlying non-HA resource problems.

Any thoughts on that?

**Comment by jimaek** [comment]
The problem is not the CDN itself. We dont host the actual file, we proxy and cache it from @polyfills official server. That server went down and caused the issue. The CDN operates just fine and has lots of advanced techniques of load balancing

**Comment by jonathanong** [comment]
yup it's an issue with my server. haven't touched it in a while

**Comment by nilskp** [comment]
Yeah, I understand that part. But I think we can agree that the observed behavior is sub-optimal. In this case, it would seem better to keep serving the cached file, if the server isn't reachable. Is that a configuration thing or what's the way to achieve that?

**Comment by jimaek** [comment]
Our agreement was to proxy everything as is from the origin server. I now changed the config to prevent updating the cache if the server has issues. It should help but its not bulletproof, if the server will go down for days nothing will save it.

**Comment by nilskp** [comment]
@jonathanong Any ETA on this?

**Comment by jonathanong** [comment]
seems that one of the polyfills changed their package structure

**Comment by jimaek** [comment]
@jonathanong Your server is still down. Any ideas?

**Comment by jonathanong** [comment]
Yeah one of the libraries messed things up. Haven't updated it. Trying to get more collaborators...

**Comment by evandrix** [comment]
I'm confused. Given that this service is down, which repositories do I need exactly in order to mirror & host it myself?

**Comment by jonathanong** [comment]
these are the relevant repos:

https://github.com/polyfills/db
https://github.com/polyfills/feature
https://github.com/polyfills/polyfill-ua

this is the actual server: https://github.com/polyfills/paas

let me know if you wanna help

**Comment by evandrix** [comment]
When will https://cdn.jsdelivr.net/polyfills/polyfill.js be restored?

**Comment by brad-jones** [comment]
Man I got so excited when I saw this project in a google search, and then so disappointed when I eventually got the 500 error. I think it has some merit. Any update on the status of the project?

**Comment by jaimeagudo** [comment]
Still experiencing the 500 upon GET https://cdn.jsdelivr.net/polyfills/polyfill.js :(
